package.json

```diff
@@ -10,6 +10,7 @@
     }
   ],
   "devDependencies": {
+    "axios": "^0.15.3",
```

Comment:
That's a Bower issue, we are not using bower, and the discussion over dev or not direct is a matter of opinion. In mine I think that everything that's required run-time should be in "dependencies" otherwise in "devDependencies"